components/InfoCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { Card, CardActionArea, CardContent } from '@mui/material'
+import Router from 'next/router'
+import React from 'react'
+
+function InfoCard({ userInfo, onClickRoute, idPrefix }) {
```

Comment:
More general names for the parameters, for example  instead of `onClickRoute` you can use only `onClick` because the action does not mean a different route